schemas/WebpackOptions.json

```diff
@@ -1564,6 +1564,7 @@
             "javascript/dynamic",
             "javascript/esm",
             "json",
+            "url",
```

Comment:
To begin with I would start with `url/experimental` and remove the default rules, so this feature is opt-in.